app/scripts/components/common/form/checkable-filter/index.tsx

```diff
@@ -90,7 +91,7 @@ export default function CheckableFilters(props: CheckableFiltersProps) {
     globallySelected,
     tagItemCleared
   } = props;
-  const [show, setShow] = useState<boolean>(true);
+  const [show, setShow] = useState<boolean>(false);
```

Comment:
If I remember correctly, we decided that the filter menu should be open by default.

Reply:
That change is coming from @faustoperez above: https://github.com/NASA-IMPACT/veda-ui/pull/989#pullrequestreview-2093840574

Reply:
Oh but I think it is possible that @faustoperez meant it only for the Exploration page Modal. Can you clarify? @faustoperez

Reply:
> Oh but I think it is possible that @faustoperez meant it only for the Exploration page Modal

Yes, only on the E&A page modal. Space is tighter and filters should be collapsed there 👍